test_my_gspread.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+import gspread
+import pandas as pd
+from pandas import json_normalize
+from dotenv import load_dotenv
+import os
+import requests
+import json
+import datetime as dt
+
+from my_gspread import get_or_create_worksheet
+from my_gspread import pass_json_into_df
+
+load_dotenv()
+gspread_key = os.getenv('Project_key')
```

Comment:
[Don't Repeat Yourself (DRY)](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

This is something you did in the `my_gspread.py` already. Why not create a setup() or connect_sheet() that you can always call?